packages/cashier/src/types/types.ts

```diff
@@ -0,0 +1,14 @@
+export type TServerError = {
```

Comment:
I think it's better to move this file to the `types/shared` or `types/stores` folder because these types will be used for other stores, too.

Reply:
@hamid-deriv I see we create a separate file for each type, I'm not sure why or if I should do the same. I will move the `types.ts` file under `types/shared` for now 🙇🏻